pkg/controller/node/controller_utils.go

```diff
@@ -263,12 +263,12 @@ func nodeExistsInCloudProvider(cloud cloudprovider.Interface, nodeName types.Nod
 	return true, nil
 }
 
-func recordNodeEvent(recorder record.EventRecorder, nodeName, nodeUID, eventtype, reason, event string) {
+func recordNodeEvent(recorder record.EventRecorder, nodeName, namespace, nodeUID, eventtype, reason, event string) {
 	ref := &clientv1.ObjectReference{
 		Kind:      "Node",
 		Name:      nodeName,
 		UID:       types.UID(nodeUID),
-		Namespace: "",
+		Namespace: namespace,
```

Comment:
@knightXun `Node` is a non-namespaced object, so we don't need namesapce info in nodecontroller event. It should always be empty.